lib/Module.js

```diff
@@ -58,6 +58,7 @@ class Module extends DependenciesBlock {
 		this.providedExports = null;
 		this._chunks.clear();
 		this._chunksDebugIdent = undefined;
+		this.optimizationBailout = [];
```

Comment:
Could you clear it by setting `length = 0`? This should avoid allocating many new objects.

Reply:
Done